components/scream/src/share/io/scorpio_output.cpp

```diff
@@ -26,7 +26,7 @@ void AtmosphereOutput::init()
   m_is_restart = m_params.get<bool>("RESTART FILE",false);  // optional, default to false
 
   // Gather data from grid manager:  In particular the global ids for columns assigned to this MPI rank
-  EKAT_REQUIRE_MSG(m_grid_name=="Physics","Error with output grid! scorpio_output.hpp class only supports output on a Physics grid for now.\n");
+  EKAT_REQUIRE_MSG(m_grid_name=="Physics" || m_grid_name=="Physics GLL","Error with output grid! scorpio_output.hpp class only supports output on a Physics grid for now.\n");
```

Comment:
Might it be helpful to explicitly list the valid "Physics grids" in this error message?

Reply:
Good suggestion, I've made the change.